scripts/order_water.js

```diff
@@ -114,7 +120,16 @@ const handlerCommunication = (robot, queries) => {
   });
 };
 
+const waterCommands = [
+  '@mona water',
+  '@mona order water',
+  '@mona order water, please',
+  '@mona, water',
+  '@mona, order water',
+  '@mona, order water, please',
+];
+
 // This one triggers ONLY on those phrases
 module.exports = (robot) => {
-  handlerCommunication(robot, ['@mona закажи воду', '@mona order water']);
+  handlerCommunication(robot, waterCommands);
```

Comment:
Can we just use a regular expression (or list of regexes) here instead of a list of exact commands? It's annoying when you just add a dot at the end of sentence and bot can't understand you.